🗃(learninglocker) set mongodb 4.0 as the default database version

sampaccoud opened the pull request:
## Purpose

Latest versions of Learning Locker require MongoDB v4.0

## Proposal

- [x] Change default database version in the learninglocker app to v4.0
- [x] upgrade learninglocker to 5.2.2 and xapi-service to 2.9.10

sampaccoud committed:
🗃(learninglocker) set mongodb 4.0 as the default database version

latest version of Learning Locker require Mongodb v4.

lunika reviewed (approved):
![](https://media.giphy.com/media/fu8SPp0DMFtr8jCOrx/giphy.gif)

sampaccoud committed:
📌(learninglocker) upgrade to learninglocker v5.2.2 and xapi v2.9.10

We configured MongoDB v4.0 as the default database so we should upgrade
the application images to their latest version for coherence.

jmaupetit reviewed (approved):
![](https://media3.giphy.com/media/tIeCLkB8geYtW/giphy.gif)
